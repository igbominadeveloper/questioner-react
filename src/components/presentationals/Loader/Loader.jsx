import React from 'react';

import Spinner from '../../../assets/images/loader.svg';
const Loader = () => {
  return <img src={Spinner} />;
};

export default Loader;
