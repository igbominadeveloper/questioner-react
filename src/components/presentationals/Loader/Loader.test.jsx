import React from 'react';
import { shallow } from 'enzyme';

import Loader from './Loader';

describe('Loader', () => {
  describe('Render', () => {
    it('renders without error', () => {
      const component = shallow(<Loader />);
      expect(component).toMatchSnapshot();
    });
  });
});
