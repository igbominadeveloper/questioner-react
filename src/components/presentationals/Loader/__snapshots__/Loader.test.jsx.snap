// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Render renders without error 1`] = `
<div
  className="loader-overlay"
>
  <img
    className="loader"
    src=""
  />
</div>
`;
